cnf-certification-test/webserver/index.html

```diff
@@ -9,6 +10,9 @@
           sizes="any"
           href="https://ux.redhat.com/assets/logo-red-hat.svg">
     <link rel="stylesheet" href="https://ux.redhat.com/assets/packages/@rhds/elements/elements/rh-footer/rh-footer-lightdom.css">
+    <script src="https://ajax.googleapis.com/ajax/libs/jquery/2.1.1/jquery.min.js"></script>
+   <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.0.13/css/all.css" integrity="sha384-DNOHZ68U8hZfKXOrtjWvjxusGo9WQnrNx2sqG0tfsghAvtVlRW3tvkXWZh58N9jp" crossorigin="anonymous">
```

Comment:
Font-Awesome's latest release is [v6.4.2](https://github.com/FortAwesome/Font-Awesome/releases/tag/6.4.2).